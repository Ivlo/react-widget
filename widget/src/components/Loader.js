import React from "react";

const Loader = ({ text }) => {
  return <p>{text}</p>;
};

export default Loader;
