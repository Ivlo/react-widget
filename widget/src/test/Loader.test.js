import React from "react";
import { shallow } from "enzyme";
import Loader from "../components/Loader";

const textLoader = "Loading...";
const wrapper = shallow(<Loader text={textLoader} />);

describe("Loader", () => {
  it("should render the Loader Component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("should render with a paragraph equal to textLoader", () => {
    expect(wrapper.find("p").text()).toEqual(textLoader);
  });
});
