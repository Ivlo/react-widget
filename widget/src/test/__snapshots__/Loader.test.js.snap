// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render the Loader Component correctly 1`] = `
<p>
  Loading...
</p>
`;
